cranelift-bforest/Cargo.toml

```diff
@@ -14,6 +14,11 @@ edition = "2018"
 [dependencies]
 cranelift-entity = { path = "../cranelift-entity", version = "0.44.0", default-features = false }
 
+[features]
+default = ["std"]
```

Comment:
:(

1. cranelift-bforest doesn't have any extra functionality in std mode, so could you instead change the test script to not pass the core feature.
2. Please use `default-features = false` when depending on this or the other crates with `no_std` support. Otherwise depending on for example cranelift-codegen would enable `std` even when in `no_std` mode. I had removed `default-features = false` in https://github.com/CraneStation/cranelift/pull/1087, as I removed the `std` feature.

Reply:
Ok, I'll admit I didn't quite know what i was doing. Let's move this to a separate branch/PR.